lib/config/presets/internal/regex-managers.ts

```diff
@@ -44,6 +44,21 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  mavenPropertyVersions: {
+    description: 'Update `*.version` properties in `pom.xml` files.',
+    regexManagers: [
+      {
+        customType: 'regex',
+        datasourceTemplate:
+          '{{#if datasource}}{{{datasource}}}{{else}}maven{{/if}}',
+        fileMatch: ['^pom\\.xml$'],
+        matchStrings: [
+          '<!--\\s?renovate:( datasource=(?<datasource>.*?))? depName=(?<depName>.*?)\\s?(?: packageName=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s+-->\\s+<.*?\\.version>(?<currentValue>.*?)<\\/.*?\\.version>',
```

Comment:
```suggestion
          '<!--\\s?renovate:( datasource=(?<datasource>[a-z-.]+?))? depName=(?<depName>[^\\s]+?)(?: packageName=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s+-->\\s+<.+\\.version>(?<currentValue>.+)<\\/.+\\.version>',
```

* `datasource`: other regex manager presets also limit it to `[a-z-.]+?`
  * https://github.com/renovatebot/renovate/pull/24001
* `depName`: same format as `packageName`, so should be matched analogously
* `<.version></.version>` would also be matched but this is not legit 
  * refine to `<.+\.version>(?<currentValue>.+)<\/.+\.version>`